### ISSUE TITLE
issue about kernel_bilinear_cnn_all.py

### ISSUE THREAD

**Issue by Yichan16** [xueluli/Kernel-Pooling#1]
Hi, thanks for your codes first. I note that in the 93-94 lines of "kernel_bilinear_cnn_all.py" (param.requires_grad = False), the parameters of VGG-16 are set to be unable to learn. Whether should it be set to True?

But after modifying param.requires_grad = True, I got a "nan" loss. and the accuracy is always 0.52%.

Could you help me and tell me something I should pay attention to ? Thanks a lot.

**Comment by xueluli** [comment]
Hi, sorry for the mistake. You can just comment out the part. The parameters need to be learned. I did not obtain a  "nan" loss using  the codes. However, pay attention to the values of a0, a1, ..., a4, which influence the performance a lot. I forget whether the values listed in the code is the final choice, you may refer the original paper for more details about how to choose the values.

**Comment by xueluli** [comment]
> Hi, thanks for your codes first. I note that in the 93-94 lines of "kernel_bilinear_cnn_all.py" (param.requires_grad = False), the parameters of VGG-16 are set to be unable to learn. Whether should it be set to True?
> 
> But after modifying param.requires_grad = True, I got a "nan" loss. and the accuracy is always 0.52%.
> 
> Could you help me and tell me something I should pay attention to ? Thanks a lot.


Hi, sorry for the mistake. You can just comment out the part. The parameters need to be learned. I did not obtain a "nan" loss using the codes. However, pay attention to the values of a0, a1, ..., a4, which influence the performance a lot. I forget whether the values listed in the code is the final choice, you may refer the original paper for more details about how to choose the values.
